docs/configuration_options.rst

```diff
@@ -88,6 +88,16 @@ Data Conversion
   * Default: false
   * Importance: low
 
+``compact.map.entries``
+  Defines how map entries with string keys within record values should be written to JSON. When this is set to ``true``, these entries are written compactly as ``\"entryKey\": \"entryValue\"``.
```

Comment:
in the `.rst` the double quotes `"` need to be unescaped. (same as when you run `ElasticsearchSinkConnectorConfig.main`

Reply:
ack